airflow/contrib/operators/azure_container_instances_operator.py

```diff
@@ -107,9 +107,24 @@ class AzureContainerInstancesOperator(BaseOperator):
     template_fields = ('name', 'environment_variables')
 
     @apply_defaults
-    def __init__(self, ci_conn_id, registry_conn_id, resource_group, name, image, region,
-                 environment_variables=None, volumes=None, memory_in_gb=None, cpu=None,
-                 command=None, remove_on_error=True, fail_if_exists=True, *args, **kwargs):
+    def __init__(
```

Comment:
There is a change in sequence of positional parameters and it's likely someone is using it already. I am not sure this is a good idea to change it in (I assume) already released operator?